Assets/MRTK/Providers/Oculus/XRSDK/OculusXRSDKDeviceManager.cs

```diff
@@ -33,6 +34,15 @@ public class OculusXRSDKDeviceManager : XRSDKDeviceManager
             uint priority = DefaultPriority,
             BaseMixedRealityProfile profile = null) : base(inputSystem, name, priority, profile) { }
 
+        public override void Initialize()
+        {
+            base.Initialize();
+#if !OCULUSINTEGRATION_PRESENT && UNITY_EDITOR && UNITY_ANDROID
```

Comment:
nit: should the whole `Initialize()` be wrapped in this? since it doesn't do anything otherwise